mne/preprocessing/tests/test_beer_lambert_law.py

```diff
@@ -0,0 +1,35 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+from mne.datasets.testing import data_path
+from mne.io import read_raw_nirx, BaseRaw
+from mne.preprocessing import optical_density, beer_lambert_law
+from mne.utils import _validate_type
+from mne.datasets import testing
+
+
+fname_nirx = op.join(data_path(download=False),
+                     'NIRx', 'nirx_15_2_recording_w_short')
+
+
+@testing.requires_testing_data
+def test_beer_lambert():
```

Comment:
You should `pytest.mark.parametrize('fname', ...)` over the two fnames you just added. And also over round-trip I/O (FIF and NIRX native reading support). So something like:
```
pytest.mark.parametrize('fname', (...))
pytest.mark.parametrize('fmt', ('nirx', 'fif'))
def test_beer_lambert(fname, fmt, tmpdir):
    assert fmt in ('nirx', 'fif')
    raw = mne.io.read_raw_nirx(fname_nirx)
    if fmt == 'fif':
        raw.save(tmpdir.join('test_raw.fif'))
        raw = mne.io.read_raw_fif(tmpdir.join('test_raw.fif'))
    ...
```
That way we check that everything works for files that have undergoing round-trip I/O.

Reply:
Neat, I like this